### PR TITLE
Fix warnings on Xcode 9.3

### PR DESCRIPTION
**Pull request by dfed**
Warnings are bad, mkay?

### CODE REVIEW

File: Sources/Internal/Keychain.swift [hunk 1]
```diff
@@ -202,11 +202,18 @@ internal final class Keychain {
                 return SecItem.DataResult.success(Set([singleKey]))
                 
             } else if let multipleMatches = collection as? [[String: AnyHashable]] {
+                #if swift(>=4.1)
```
Comment (NickEntin) [inline Sources/Internal/Keychain.swift]: Shouldn't this always build with Swift 4.0 based on the `.swift-version` file?

Comment (dfed) [reply Sources/Internal/Keychain.swift]: If only that's how `.swift-version` worked. All that does is let Cocoapods decide that two projects are on different swift major version and therefore won't work well together.

Comment (NickEntin) [reply Sources/Internal/Keychain.swift]: I thought it also affected the `SWIFT_VERSION` in the `Pods.xcodeproj` build settings?

Comment (dfed) [reply Sources/Internal/Keychain.swift]: It may. But if you use submodules, you'll get a build warning. So we should fix this :p

Comment (NickEntin) [reply Sources/Internal/Keychain.swift]: Ahh right, good point.

**Pull request merged**